client.go

```diff
@@ -157,8 +171,10 @@ func (c *Client) listen() {
 				c.broker.sendAll(c.broker.clients[app.route], clearStateMsg)
 			}
 		case watchMsgT:
-			c.subscribe(m.addr) // subscribe even if page is currently NA
-
+			if c.isReconnect {
```

Comment:
Maybe `isReconnect` is redundant and I could use `c.cancel != nil`?

Reply:
I think using `isReconnect` is the right call - imo it makes it clearer what's going on and in the future debugging a bit easier